demo/run.sh

```diff
@@ -19,35 +19,34 @@ function main() {
   createDemoEnvironment
   waitForServer
   createHostFactoryToken
-  runAnsible
-  retrieveSecretInTarget
+  conjurizeTargetContainer
+  RetrieveSecretInTargetWithCli
```

Comment:
Yay we already have 3 tests cases for molecule! We should roll that in sooner than later and get real CI going. We're going to need that all buttoned up before we ship a new version.

https://github.com/metacloud/molecule

I haven't used molecule before, but there should be a way to get it going in docker-compose.